docs/UserGuide.md

```diff
@@ -43,144 +43,351 @@ By : `Team SE-EDU`  &nbsp;&nbsp;&nbsp;&nbsp; Since: `Jun 2016`  &nbsp;&nbsp;&nbs
 
 ### 2.1. Viewing help : `help`
 
-Format: `help`
+Format: `help [COMMAND]`
 
-> Help is also shown if you enter an incorrect command e.g. `abcd`
+> * Command is optional.
+> * When there is no command specified, help will print help messages of all commands.
+> * Help is also shown if you enter an incorrect command e.g. `abcd`
 
-### 2.2. Adding a person: `add`
+### 2.2. Adding an event/deadline/task : `add`
 
-Adds a person to the address book<br>
-Format: `add NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...`
+#### 2.2.1. Adding an event : `add ev`
 
-> Persons can have any number of tags (including 0)
+Adds an event to the event list. Undoable.<br>
+Format: `add ev DESCRIPTION [s/START_TIME] f/START_DATE [e/END_TIME] [t/END_DATE] [c/COMMENTS]... [l/LOCATION]`
```

Comment:
Try to stay away from command line switches if possible. Make the command as flexible as you can.